pkg/virtctl/vnc/vnc.go

```diff
@@ -70,6 +74,9 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 			return c.Run(cmd, args)
 		},
 	}
+	cmd.Flags().BoolVar(&proxyOnly, "proxy-only", proxyOnly, "--proxy-only=false: Setting this true will run only the virtctl vnc proxy and show the localhost port where VNC viewers can connect")
+	cmd.Flags().IntVar(&customPort, "custom-port", customPort,
+		"--custom-port=0: Assigning a port value to this will try to run the proxy on the given port if the port is accessible; If unassigned, the proxy will run on a random port")
```

Comment:
would `--port ` and `-p` make sense?

Reply:
@rmohr Yes, changed it to just `--port` now